build.gradle

```diff
@@ -243,6 +280,7 @@ project(':iceberg-data') {
     compile project(path: ':iceberg-core', configuration: 'shadow')
     compileOnly project(path: ':iceberg-parquet', configuration: 'shadow')
     compileOnly project(path: ':iceberg-orc', configuration: 'shadow')
+    compileOnly project(path: ':iceberg-common', configuration: 'shadow')
```

Comment:
If `iceberg-data` didn't need `iceberg-common` before why does it now?

Reply:
And this?

Reply:
iceberg-common can be removed from here

Reply:
Please do the necessary!

Reply:
I was wrong about this, double checked and tests fail if I don't include `iceberg-common`